src/dev/config.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// This file is strictly used for private development
+// Nothing in this file should have a truthy value on mainline
+
+export const betaUrl = ''
```

Comment:
Future work: replace this file with a top-level 'config.json' that is embedded into the build, setting the default values used by `DevSettings`.

Reply:
could it be config.ts ? That way we get autocomplete etc.

Reply:
Oh I meant `config.ts`